support-payment-api/src/test/scala/conf/ConfigLoaderProvider.scala

```diff
@@ -15,8 +15,9 @@ trait ConfigLoaderProvider extends BeforeAndAfterAll { self: Suite =>
   private val ssm: AWSSimpleSystemsManagement = AWSClientBuilder.buildAWSSimpleSystemsManagementClient()
   private val configLoader = new ConfigLoader(ssm)
 
-  def configForTestEnvironment[A: ParameterStoreLoadable[Environment, ?]: ClassTag](): A =
+  def configForTestEnvironment[A: ParameterStoreLoadable[Environment, *]: ClassTag](): A = {
```

Comment:
Corrected for  errors
 1.conf.ConfigLoader.ParameterStoreLoadable[model.Environment, _] does not take type parameters
2.could not find implicit value for evidence parameter of type conf.ConfigLoader.ParameterStoreLoadable[model.Environment,A]